less/mosaico/article.less

```diff
@@ -332,7 +307,7 @@
     }
   }
   &__subheadline {
-    font: 600 1rem/1.5rem "Roboto", sans-serif;
+    font: 500 1.07rem/1.5rem "Roboto", sans-serif;
```

Comment:
Can the `rem` be removed to make this like line 232 above?

Reply:
Made a Trello card for fixing the line-height as there are several places where rem is used